package.json

```diff
@@ -19,40 +31,27 @@
     "karma-sourcemap-writer": "^0.1.2",
     "karma-spec-reporter": "0.0.26",
     "karma-transform-path-preprocessor": "0.0.3",
-    "karma-webpack": "^1.7.0"
-  },
-  "dependencies": {
-    "@angular-redux/store": "^6.0.0",
-    "@angular/compiler": "^2.4.2",
-    "@angular/compiler-cli": "^2.4.2",
-    "@angular/core": "^2.4.2",
-    "@angular/forms": "^2.4.2",
-    "@angular/tsc-wrapped": "^0.5.1",
+    "karma-webpack": "^1.7.0",
+    "@angular-redux/store": "^6.1.0-alpha.1",
     "@ngtools/webpack": "^1.2.1",
-    "@types/chai": "^3.4.34",
     "@types/jasmine": "^2.5.35",
     "@types/node": "^6.0.45",
-    "babel-core": "^6.11.4",
-    "babel-loader": "^6.2.5",
-    "babel-preset-es2015": "^6.14.0",
```

Comment:
Making our consumers depend on babel is not super nice given that that's not the norm in the Typescript world.